tools/scripts/run_code_generation.py

```diff
@@ -456,6 +535,15 @@ def main():
 
     highly_refined_percent_of_cores_to_take = 0.9
     max_workers = max(1, int(highly_refined_percent_of_cores_to_take * os.cpu_count()))
+    
+    #build reverse map
+    # Open a file using 'with' and read its contents
+    smithy_c2j_data = {}
+    c2j_smithy_data = {}        
+    with open(os.path.abspath(SMITHY_TO_C2J_MAP_FILE), 'r') as file:
+        smithy_c2j_data = json.load(file)
+        # Reverse the key-value pairs
+        c2j_smithy_data = {value: key for key, value in smithy_c2j_data.items()}
```

Comment:
I can see you are reversing a map from the code
I'd prefer to have a comment explaining why

Reply:
ack